src/Cache/Context/OgMembershipStateCacheContext.php

```diff
@@ -82,25 +69,15 @@ public static function getLabel() {
    * {@inheritdoc}
    */
   public function getContext() {
-    if (!$route_contexts = $this->routeMatch->getRouteObject()->getOption('parameters')) {
-      // No "parameters" defined in the route.
-      return self::NO_CONTEXT;
-    }
-
-    if (!$entity_type_ids = array_keys($this->groupTypeManager->getAllGroupBundles())) {
-      // No group entities.
-      return self::NO_CONTEXT;
-    }
-
-    if (!$entity_type_ids = array_intersect(array_keys($route_contexts), $entity_type_ids)) {
-      // No parameters that match the group entities.
+    // Do not provide a cache context if there are no groups in the current
+    // context.
+    $contexts = $this->ogContext->getRuntimeContexts(['og']);
```

Comment:
yeah, I was just not clear on what _other_ value you might have there apart of `['og']` (so maybe we can have a helper method to call `getRuntimeContexts(['og']);` since that's what we'll normally use, no?). 

Anyway, looks good